common/src/main/java/io/netty/util/collection/PrimitiveCollections.java

```diff
@@ -122,9 +142,12 @@ public boolean containsValue(Object value) {
      *
      * @param <V> the value type stored in the map.
      */
-    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V>,
-            Iterable<IntObjectMap.Entry<V>> {
+    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V> {
         final IntObjectMap<V> map;
+        Set<Integer> keySet;
```

Comment:
The class visibility and the member visibility are independent.  Unless you are using these members outside the scope of the inner class (intentionally package private to avoid the potential of synthetic method generate) I think we should set the accessor level like we would for regular classes.